Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG SC_PYTHON_VERSION=3.13
+
+FROM python:${SC_PYTHON_VERSION}-slim-bookworm AS local
+
+ARG SC_PYTHON_VERSION=3.13
```

Comment:
duplicate of line 1?

Reply:
also is this needed if it's defined in `docker-compose.yml`?

Reply:
Unfortunately ARG is stage scoped so since I want to start with python base but also reference the version in a COPY command it has to be duplicated. I'll add a comment explaining this though.